src/Domain/CoordinateMaps/TimeDependent/ShapeMapTransitionFunctions/SphereTransition.cpp

```diff
@@ -12,12 +12,13 @@
 #include "Utilities/ContainerHelpers.hpp"
 #include "Utilities/EqualWithinRoundoff.hpp"
 #include "Utilities/ErrorHandling/Error.hpp"
+#include "Utilities/MakeString.hpp"
 
 namespace domain::CoordinateMaps::ShapeMapTransitionFunctions {
 
 SphereTransition::SphereTransition(const double r_min, const double r_max,
-                                   const bool reverse)
-    : r_min_(r_min), r_max_(r_max) {
+                                   const bool reverse, const bool extend)
```

Comment:
Just enable this always, can't hurt. That's what other maps do.

Reply:
The reason I have the choice and explicitly don't enable it in most cases is because this does impact control system horizon finds. This is different from other maps because the shape map acts in the distorted frame. If we always allow this to succeed, then when finding the horizon in the distorted frame and a point moves outside a block with a distorted frame, the inverse (of a different block) would still find the point and we'd never know that the horizon has actually moved out of a region with a distorted frame.

I think it's better for keeping track of things to only allow an extended shape map if we actually need it

Reply:
I thought we find horizons only in blocks that have a distorted frame, meaning we don't even try the inverse when there is no distorted frame. Also in the logical frame we usually reject points outside [-1, 1].

To allow extrapolation into excisions we need the map extension to be enabled in domains written to H5 files. I think the solution is to decide whether to allow or reject logical coords outside [-1, 1] after calling the map, not make the map valid or invalid outside those bounds at construction.